etc/env/ext-node.toml

```diff
@@ -2,8 +2,9 @@
 # All the variables must be provided explicitly.
 # This is on purpose: if EN will accidentally depend on the main node env, it may cause problems.
 
-test_database_url = "postgres://postgres@localhost:5433/zksync_local_test_ext_node"
 database_url = "postgres://postgres@localhost/zksync_local_ext_node"
+template_database_url="postgres://postgres@localhost/zksync_local"
```

Comment:
Probably it makes sense to add a comment on this variable, e.g. "Optional variable. If set, `zk` tool will use this URL to dump the database contents. Not used by EN itself".

Reply:
done